game/src/edit/lanes.rs

```diff
@@ -131,6 +140,10 @@ impl SimpleState<App> for LaneEditor {
                 app,
                 app.primary.map.get_l(self.l).parent,
             )),
+            "Modify entire road" => Transition::Push(crate::edit::roads::prompt_for_lanes(
```

Comment:
Just to get on the same page - eventually the "modify entire road" UI will supplant the current per-lane editing, and we'd just get rid of the per-lane editing all together, right?

Reply:
Correct. As Yuwen's design currently stands, the change is even more dramatic -- we won't even be selecting individual lanes in sandbox or edit mode anymore, to start with. Always the entire road.